protocol/activity_center_persistence.go

```diff
@@ -610,11 +644,39 @@ func (db sqlitePersistence) DismissAllActivityCenterNotifications() error {
 }
 
 func (db sqlitePersistence) DismissAllActivityCenterNotificationsFromUser(userPublicKey string) error {
-	_, err := db.db.Exec(`UPDATE activity_center_notifications SET read = 1, dismissed = 1 WHERE NOT dismissed AND NOT accepted AND author = ?`, userPublicKey)
+	_, err := db.db.Exec(`
+		UPDATE activity_center_notifications
+		SET read = 1, dismissed = 1
+		WHERE NOT deleted
```

Comment:
Similarly, we could check indexes here, just to make sure

Reply:
Similar results from the other thread @cammellos. Currently there's no index covering the query (even in `develop` branch).

Running the following statements solves the problem.

```sql
CREATE INDEX activity_center_deleted_dismissed_accepted_author_idx
  ON activity_center_notifications(deleted, dismissed, accepted, author);

EXPLAIN QUERY PLAN
UPDATE activity_center_notifications
  SET read = 1, dismissed = 1
  WHERE NOT deleted AND NOT dismissed AND NOT accepted AND author = 'somebody';

-- Result:
-- SCAN activity_center_notifications USING COVERING INDEX activity_center_deleted_dismissed_accepted_author_idx
```